packages/common/driver-definitions/src/storage.ts

```diff
@@ -161,6 +161,11 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
 	// eslint-disable-next-line @rushstack/no-new-null
 	getSnapshotTree(version?: IVersion, scenarioName?: string): Promise<ISnapshotTree | null>;
 
+	getSnapshot?(
+		version?: IVersion,
```

Comment:
do we ever use version? I'd think we do not, and SPO does not support it anyway. If so, should it be removed?

Reply:
Ah, I now remember that our test code (summarization tests) assume they can load from old version. I think these tests are only done on local server or something like that. So I guess we have to have it (even though it's not implemented by ODSP).
Should it move into snapshotFetchOptions? It's weird that we use one arg explicitly, and the rest is passed in a structure.

Reply:
Moved it to snapshotFetchOptions